Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM python:3.8-slim
+FROM python:3.9-slim
 
 # Install dependencies for OpenCV
-RUN apt-get update && apt-get -y install libgl1 libglib2.0-0 git
+RUN apt-get update && apt-get -y install libgl1 libglib2.0-0 git gcc
```

Comment:
We do not want to add gcc as part of the build process. Can you find a different way to accomplish updating packages for 3.9?

Reply:
This whole line can be removed (no `apt` packages in the docker image) by bumping d.py to 1.7 and changing to `opencv-python-headless`.

This saves ~200MB in the final image.

Reply:
Unfortunately we are using discord 1.5.1 which has dependencies that have no wheels, If we update discord.py to 1.7.3 (latest) we should be able to remove gcc as part of the build process.